Framework/Algorithms/test/PolarizationEfficiencyCorTest.h

```diff
@@ -486,10 +427,35 @@ class PolarizationEfficiencyCorTest : public CxxTest::TestSuite {
     return names;
   }
 
+  std::shared_ptr<PolarizationEfficiencyCor> createAlgorithm() {
+    auto alg = std::make_shared<PolarizationEfficiencyCor>();
+    alg->setRethrows(true);
+    alg->initialize();
+    return alg;
+  }
+
+  void setAlgorithmProperties(const std::shared_ptr<PolarizationEfficiencyCor> &alg,
+                              const std::string &efficiencyMethod, const std::string &method = "",
+                              const std::string &analysisMethod = "") {
+    alg->setProperty("OutputWorkspace", "out");
+    if (!method.empty()) {
+      alg->setProperty("CorrectionMethod", method);
+    }
+    alg->setProperty("Efficiencies", createEfficiencies(efficiencyMethod));
+    if (!analysisMethod.empty()) {
+      alg->setProperty("PolarizationAnalysis", analysisMethod);
+    }
+  }
+
+  void checkWorkspaceGroupSize(size_t expectedSize) {
```

Comment:
This method is checking the size of a specific workspace group (i.e. the one called "out"). Would it be worth renaming it so that it's called `checkOutputWorkspaceGroupSize` to make that clearer?